.circleci/config.yml

```diff
@@ -637,3 +646,5 @@ workflows:
           parallelism: 26
           requires:
             - build-sandboxes
+
+# VS Code Extension Version: 1.5.0
```

Comment:
Was this line intentional @ndelangen ?

Reply:
no!, how did this happen?

Reply:
fixed here:
https://github.com/storybookjs/storybook/pull/20510